packages/twenty-front/src/modules/views/hooks/useAreViewFilterGroupsDifferentFromRecordFilterGroups.ts

```diff
@@ -0,0 +1,59 @@
+import { currentRecordFilterGroupsComponentState } from '@/object-record/record-filter-group/states/currentRecordFilterGroupsComponentState';
+import { useRecoilComponentValueV2 } from '@/ui/utilities/state/component-state/hooks/useRecoilComponentValueV2';
+import { useGetCurrentViewOnly } from '@/views/hooks/useGetCurrentViewOnly';
+import { getViewFilterGroupsToCreate } from '@/views/utils/getViewFilterGroupsToCreate';
+import { getViewFilterGroupsToDelete } from '@/views/utils/getViewFilterGroupsToDelete';
+import { getViewFilterGroupsToUpdate } from '@/views/utils/getViewFilterGroupsToUpdate';
+import { mapRecordFilterGroupToViewFilterGroup } from '@/views/utils/mapRecordFilterGroupToViewFilterGroup';
+import { useMemo } from 'react';
+import { isDefined } from 'twenty-shared';
+
+export const useAreViewFilterGroupsDifferentFromRecordFilterGroups = () => {
+  const { currentView } = useGetCurrentViewOnly();
+
+  const currentRecordFilterGroups = useRecoilComponentValueV2(
+    currentRecordFilterGroupsComponentState,
+  );
+
+  const viewFilterGroupsAreDifferentFromRecordFilterGroups = useMemo(() => {
```

Comment:
is this useMemo useful?

This hook is taking dependecy on currentView (prefetchViewFromViewIdFamilySelector behind the scene) + currentRecordFilterGroups which seems to be perfect

Reply:
I can remove it yes